R/calcEmissions4ReportExtra.R

```diff
@@ -0,0 +1,18 @@
+#' Calculates historical emissions needed for estimating emission factors
+#' in extra emissions reporting
+#'
+#' @param source either "PBL_2007" or "PBL_2022"
```

Comment:
there is no source parameter

Reply:
Ops, my bad. Fixed that, improved the documentation a bit and reran `make docs` now.